puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/deploy/i18n-string-extraction-and-update.groovy

```diff
@@ -0,0 +1,87 @@
+pipeline {
+  agent { label 'admin && sshkey' }
+  environment {
+  ruby_version = '2.5'
+  triggers {
+  	 TZ='Asia/Kolkata'
+  	 cron('0 10 */7 * *')
+  }
+  }
+  stages {
+    stage('Setup Environment') {
+      steps {
+        git url: 'https://github.com/theforeman/foreman', branch: 'develop'
+        script {
+          sh 'git checkout -b i18n-updates'
+          configureRVM(ruby_version)
+          withRVM(['bundle install --jobs=5 --retry=5'], ruby_version)
+          sh 'cp config/database.yml.example config/database.yml'
+        }
+      }
+    }
+    stage('Extract strings and update translations'){
+      steps {
+        script {
+          withCredentials([string(credentialsId: 'transifex-api-token', variable: 'TX_TOKEN')]) {
+            sh 'make -C locale/ tx-update'
```

Comment:
Do the make commands use the Ruby environment?

Reply:
changed this to run in RVM env